site/pages/competencies/tech-lead.html

```diff
@@ -0,0 +1,11 @@
+---
+title: Tech Lead Competencies
+description: Engineering competencies for Tech Leads.
+permalink: /competencies/tech-lead/
+layout: o-layout-docs
+---
+
+
+<h1 id="senior2">Tech Lead</h1>
```

Comment:
Not sure what the ID is used for here, but probably should match the role it covers.
```suggestion
<h1 id="tech-lead">Tech Lead</h1>
```